pyproject.toml

```
@@ -0,0 +1,173 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "finnizeclient"
+dynamic = ["version"]
+description = 'open source repository that help to sent the signal to finnize website'
+readme = "README.md"
+requires-python = ">=3.7"
+license = "MIT"
+keywords = []
+authors = []
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = []
+
+[project.urls]
+Documentation = "https://github.com/unknown/finnizeclient#readme"
+Issues = "https://github.com/unknown/finnizeclient/issues"
+Source = "https://github.com/unknown/finnizeclient"
+
+[tool.hatch.version]
+path = "src/finnizeclient/__about__.py"
```

```suggestion
path = "finnizeclient/_version.py"
```

also add `finnizeclient/_version.py` with context `__version__ = "0.1.0"`